applicationinsights.ts

```diff
@@ -22,6 +23,7 @@ class ApplicationInsights {
     * The default client.
     */
     public static client: Client;
+    public static contracts = Contracts;
```

Comment:
Not really. The only problem I had was with source maps, which is case-sensitive for file paths, but this is not the case here.